src/quick/osgviewer/view.cpp

```diff
@@ -105,6 +105,8 @@ osg::CameraQtQml *ViewQtQuick::Index::getCamera()
 void ViewQtQuick::Index::setCamera(osg::CameraQtQml *camera)
 {
     o(this)->setCamera(camera->camera());
+    o(this)->getCamera()->setPreDrawCallback(preDraw.get());
+    o(this)->getCamera()->setPostDrawCallback(postDraw.get());
```

Comment:
Ok. It's done by me.
Please review new [camera.qml](https://github.com/podsvirov/osgqtquick/blob/develop/tests/camera.qml) test.